liota/lib/utilities/utility.py

```diff
@@ -272,6 +273,16 @@ def read_liota_config(section, name):
         log.warn('liota.conf file missing')
     return value
 
+def read_user_config(config_file_path):
+
+    config = ConfigParser.RawConfigParser()
+    config.optionxform = str
+    config.read(config_file_path)
+
+    user_config = dict(config.items('DEFAULT'))
```

Comment:
If someone inserts new section tomorrow in sampleProp.conf, should we make user_config to hold data from all the sections? As this is utility method I believe we can keep it generic for our usage.

```
for each_section in conf.sections():
    for (each_key, each_val) in conf.items(each_section):
        print each_key
        print each_val
```